### PR TITLE
drone/docker: prepare multi-arch release + provide arm64 image

### PR DESCRIPTION
**Pull request by sapk**
Following #7480 this PR make the current docker release suffixed  by `-linux-amd64` and add a manifest pipeline linking the default latest and tagged version to corresponding arch.

Later we will only need to add specific docker-os-arch pipeline (+ manifest) to add a supported arch.

This PR is based on what the drone plugin does:
https://github.com/drone-plugins/drone-docker/blob/master/docker/docker/manifest.tmpl
https://github.com/drone-plugins/drone-docker/blob/master/.drone.yml#L303

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -615,6 +616,36 @@ steps:
         exclude:
         - pull_request
 
+
+---
+kind: pipeline
+name: docker-manifest
```
Comment (techknowlogick) [inline .drone.yml:683]: We also have an arm64 runner that you can target

Comment (sapk) [reply .drone.yml:683]: Thanks for the info, I added the build and manifest part for arm64.

**Comment by sapk** [comment]
I don't know why but drone doesn't like the file: https://drone.gitea.io/go-gitea/gitea/11012
I check lint via cli (drone version 1.1.2) and it doesn't report me anything.
`drone exec --event pull_request` also work.

**Comment by techknowlogick** [comment]
`notify` pipeline depends on `docker`

**Comment by sapk** [comment]
@techknowlogick Thx it works now.

**Review by techknowlogick — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**